pkg/provider/azure_controller_common.go

```diff
@@ -729,7 +729,7 @@ func getErrorCode(err error) *string {
 	return &matches[1]
 }
 
-func configAccepted(future *azure.Future) bool {
+func ConfigAccepted(future *azure.Future) bool {
```

Comment:
Consider `VMConfigAccepted` to make it clear this check a VM config update result.